package.json

```diff
@@ -18,6 +18,7 @@
     "@babel/node": "^7.8.4",
     "@babel/preset-env": "^7.8.6",
     "@babel/register": "^7.8.6",
+    "babel-eslint": "^10.1.0",
```

Comment:
@Muhire-Josue Why did you add this package?

Reply:
I used it to solve some warnings eslint was throwing about creating asyn/await functions